src/main/java/frc/robot/commands/groups/ExtendAndScoreCone.java

```diff
@@ -22,16 +21,12 @@ public ExtendAndScoreCone(DrivetrainSubsystem drive, FloorArmSubsystem floorArm,
       new WaitCommand(0.5),
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT_POSITION),
       new MoveArmToPosition(arm, Constants.ARM_SCORE_HIGH_POSITION),
-      //new WaitUntilCommand(() -> Math.abs(arm.getPosition() - arm.getTargetPosition()) < Constants.ARM_ENCODER_COUNT_ERROR),
       new WaitCommand(1),
-      new SetDriveTranslationTarget(drive, getTranslationTarget(drive, 1)),
-      new DropCone(intake).withTimeout(1),//could probably tune this down
-      new SetDriveTranslationTarget(drive, getTranslationTarget(drive, -1)),
+      new RepeatCommand(new InstantCommand(() -> drive.drive(-1, 0, 0), drive)).withTimeout(0.5),
```

Comment:
![image](https://user-images.githubusercontent.com/9126580/225155415-61b4b1d0-d90a-46b5-a201-e988d05ac01e.png)

I think something like the StartEndCommand would be ideal here, so that you can set the drive to 0